app/static/about.css

```diff
@@ -0,0 +1,56 @@
+body {
+    margin: 0;
+    padding: 0;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #F7F7F7;
+}
+
+.moving-words {
+    font-family: "Assistant", "Ariel", sans-serif;
```

Comment:
this is already set the the global.css, please connect it to your pages.

Reply:
changed!